src/chord.js

```diff
@@ -217,9 +204,9 @@ class Chord {
       bass = null,
     },
   ) {
-    this.root = root || new Key({ note: base, modifier });
     this.suffix = presence(suffix);
-    this.bass = bass || (bassBase ? new Key({ note: bassBase, modifier: bassModifier }) : null);
+    this.root = root || new Key({ note: base, modifier: modifier, minor: suffix === 'm' });
```

Comment:
Just seeing where this comes up. An option to start a discussion from my IDE popped up, gonna see what happens